best-practices/README.md

```diff
@@ -183,7 +183,9 @@ Email
 JavaScript
 ----------
 
-* Use CoffeeScript.
+* Use ES6 when possible with a tool like [6to5].
```

Comment:
I'm a fan of "Use Coffeescript, ES6 with 6to5, or another language that compiles to JavaScript.". I think it does a good enough job of saying use JavaScript but avoid ES5.